From 26daf7ee5b4a4534d918ed7d9e66d64cb30b0c26 Mon Sep 17 00:00:00 2001
From: Till Schfer <till2.schaefer@tu-dortmund.de>
Date: Tue, 22 Aug 2017 18:47:04 +0200
Subject: fix: plaintext is highlighted as powershell

Summary: see [[ https://bugs.kde.org/show_bug.cgi?id=383848 | Bug 383848 ]]

Reviewers: #framework_syntax_hightlighting, vkrause

Reviewed By: #framework_syntax_hightlighting, vkrause

Subscribers: dhaumann, #framework_syntax_hightlighting

Tags: #frameworks

Differential Revision: https://phabricator.kde.org/D7464
---
 data/syntax/powershell.xml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/syntax/powershell.xml b/data/syntax/powershell.xml
index 3a38846..2b73cfe 100644
--- a/data/syntax/powershell.xml
+++ b/data/syntax/powershell.xml
@@ -1,11 +1,10 @@
 <!DOCTYPE language SYSTEM "language.dtd">
 <language
   name="PowerShell"
-  version="1"
+  version="2"
   kateversion="5.0"
   extensions="*.ps1;*.ps1m;*.ps1d"
   section="Scripts"
-  mimetype="text/plain"
   author="Motoki Kashihara (motoki8791@gmail.com); Michael Lombardi (Michael.T.Lombardi@outlook.com)"
   casesensitive="0"
   license="LGPLv2+">
-- 
cgit v0.11.2

